submissions/kolajoTomike/Database/TransportationCompany.sql

```diff
@@ -20,27 +20,50 @@ select * from passengers;
 -- creating tables for Trip Details
 CREATE TABLE trip(
 id INT NOT NULL AUTO_INCREMENT,
-trip_id INT NOT NULL,
-registered_passengers_id INT NOT NULL,
+passengers_id INT NOT NULL DEFAULT(1),
```

Comment:
Why do you and @opeyemipeter1759  always do the same thing. You add a default value, he also did the same.

Well, This shouldn't have a default value. There is no need for that